gbin/gbin.py

```diff
@@ -99,24 +103,29 @@ def path_relative_to(self, start_path):
     @property
     def closest_venv(self):
         if not self._closest_venv:
-            inenv = self.inenv_manager or InenvManager(
-                search_start_dir=os.path.dirname(self._abs_path))
-            roots = {v['root']: k for k, v in inenv.registered_venvs.items()}
-            self.inenv_manager = inenv
-            cur_dir = os.path.dirname(os.path.dirname(self._abs_path))
-            while True:
-                if cur_dir in roots:
-                    self._closest_venv = inenv.get_venv(roots[cur_dir])
-                    return self._closest_venv
-                os.path.dirname(cur_dir)
-                next_dir = os.path.dirname(cur_dir)
-                if next_dir == cur_dir:
-                    break
-                cur_dir = next_dir
+            try:
+                inenv = self.inenv_manager or InenvManager(
+                    search_start_dir=os.path.dirname(self._abs_path))
+                roots = {v['root']: k for k, v in inenv.registered_venvs.items()}
+                self.inenv_manager = inenv
+                cur_dir = os.path.dirname(os.path.dirname(self._abs_path))
+                while True:
+                    if cur_dir in roots:
+                        self._closest_venv = inenv.get_venv(roots[cur_dir])
+                        return self._closest_venv
+                    os.path.dirname(cur_dir)
+                    next_dir = os.path.dirname(cur_dir)
+                    if next_dir == cur_dir:
+                        break
+                    cur_dir = next_dir
+            except InenvException:
+                pass
```

Comment:
This feels wrong, simply passing here will cause the property to return None unexpectedly. Also, an ignored generic error is kinda bad.